spec/requests/user_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:example) { get users_path }
+
+    it 'checks if action returns a correct response status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'checks if action rendered a correct template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'checks if correct placeholder text is shown' do
+      expect(response.body).to include('User Index')
+    end
```

Comment:
- [ ] This test failed because of the same reason I've mentioned earlier. Kindly check what you have in your users index template and include it in your test.